### PR TITLE
Update community documentation

### PR DESCRIPTION
**Pull request by owalk**
This allows github to auto detect the license for increased visability.

### CODE REVIEW

**Review by Sean-Nishi — APPROVED** [review]

**Comment by owalk** [comment]
@Sean-Nishi there was an error in a link, can you re-review when you get the chance?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#### **Did you fix whitespace, format code, or make a purely cosmetic patch?**
+
+Pull requests will not be accepted if the incoming code style is not consistent with the code style currently on the repository.
+Cosmetic changes should both match the current repository code style, and be contained to their own commit outside of the scope of functionality changes.
+
+#### **Do you intend to add a new feature or change an existing one?**
+
+* Suggest your change in the [discord server](https://github.com/m-one-w/M1WEngine?tab=readme-ov-file#Community) or in a new open issue as a feature request.
+
+* Start work on your feature, and submit a pull request when it has been completed!
```
Comment (Sean-Nishi) [inline CONTRIBUTING.md:25]: Inconsistent way of typing PR, Pull request. Choose one and keep to it?

**Review by Sean-Nishi — CHANGES_REQUESTED** [review]
Consistency when talking about PR/Pull Requests

File: CONTRIBUTING.md (hunk 1 above)
Comment (owalk) [reply CONTRIBUTING.md]: I changed all 5 instances to the full words, this needed a new commit for the instances in the readme

**Review by Sean-Nishi — APPROVED** [review]
Nice!

**Pull request merged**